.github/workflows/compressed-size.yml

```diff
@@ -23,6 +23,8 @@ jobs:
       - name: Report build size
         uses: preactjs/compressed-size-action@v2
         with:
-          build-script: 'prepack'
-          pattern: 'packages/**/dist/*.{js,cjs,mjs}'
+          build-script: 'build'
+          pattern: '{packages/**/dist/*.{js,cjs,mjs},websites/visualization/.next/static/chunks/**/*.js,websites/visualization/.next/server/pages/*.{html,json}}'
```

Comment:
Do we have way to check size of .next directory not each files?

Reply:
`compressed-size-action` checks the size of individual files, so it's probably not possible to show the overall size change of the `.next` folder.

## What I tried
- Run `compressed-size-action` as two separate scripts 
  - it didn't create two comments..



## Possible Solution
### Add a different actions to create a new comment for this
But I'm worried that this might lead to too many comments on each PR, making it hard to read.
Also, when an outsider looks at this comment, it might be difficult for them to immediately know whether we're bundling the `visualization website` or `suspensive.org`.


- [actions-next-bundle-analyzer](https://github.com/transferwise/actions-next-bundle-analyzer)
![](https://github.com/user-attachments/assets/8a59e94d-7f68-4db9-b3ef-07af88ea8d8c)
- implement it myself [reference blog](https://jeffchen.dev/posts/Measuring-Bundle-Sizes-With-Next-js-And-Github-Actions/)

What do you think I should do? If you have any good alternatives, please let me know 🙏

Reply:
> [actions-next-bundle-analyzer](https://github.com/transferwise/actions-next-bundle-analyzer)

Oh this could be good to check next built size...! could you do this for this repository?